dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -204,10 +204,14 @@ const sticky = css`
 	}
 `;
 
-const keyEventsTopMargin = css`
+const keyEventsMargins = css`
 	${from.desktop} {
 		margin-top: ${space[1]}px;
 	}
+	margin-bottom: ${space[3]}px;
+	${from.desktop} {
+		margin-bottom: 0;
+	}
```

Comment:
This one slipped in!

```suggestion
	margin-bottom: ${space[3]}px;
	${from.desktop} {
		margin-top: ${space[1]}px;
		margin-bottom: 0;
	}
```